packages/wonder-blocks-tooltip/util/active-tracker.js

```diff
@@ -0,0 +1,60 @@
+// @flow
```

Comment:
🤷‍♂️ Can we have a comment explaining what this tracker does a bit better and when we should use it?

Also: is this something that we should move out of Tooltip, would we want to use this in other components as well?

Reply:
Re: moving out of tooltip package; I wondered about that and I don't know. It wasn't super clear if anything else would need this.

I will add some more commentary though.